Tasks/PublishSymbolsV2/package.json

```diff
@@ -17,8 +17,11 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
+    "@types/node": "^18.11.9",
```

Comment:
Please downgrade node types to `16.11.39` according to the [guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode16.md#upgrading-tasks-to-node-16)

Reply:
@types/node dependency was downgraded. Thanks.